_harp/apply/_apply.md

```diff
@@ -2,7 +2,9 @@
 
   Our programmes in Nazareth and London run three times a year, starting in February, June and October.
 
-  Applications for our Summer 2017 programmes open in March.
+  We will be accepting applications for our Summer 2017 programme from Sunday 5th March 2017. Applications for the London cohort close on March 31st 2017. Applications for the Nazareth cohort close on April 30th 2017.
+
+  The summer programme will run from 26th June until 13th October 2017.
```

Comment:
This is quite wordy. I'm wondering whether we could show our dates in a clearer way?

As [this proposal](https://github.com/foundersandcoders/london-programme/issues/135#issue-197432886) was passed at the last business meeting, how about we add the list of dates just below line 3 for _all_ 2017 cohorts? It is useful for applicants to see exactly when each cohort is running, as they are always asked to mentor the next one.

We could then say

Applications dates for our Summer 2017 programmes:
```markdown
| ----- | London             | Nazareth          |
| ----- | ---------------- | ----------------- |
| Open  | Sunday 5th March  | Sunday 5th March |
| Close | Friday 31st March | Sunday 30th April  |
```